bad.py

```diff
@@ -0,0 +1,4 @@
+5 == 5
+
+concat_list = ["str1" "str2", "str3"]
+6 == 6
```

Comment:
This is always True: `6 == 6` or `6 != 6`. If testing for floating point NaN, use `math.isnan(6)`,
or `cmath.isnan(6)` if the number is complex. ([.useless-eqeq.useless-eqeq](https://semgrep.dev/r/.useless-eqeq.useless-eqeq))

<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=.useless-eqeq.useless-eqeq) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>